apps/etterlatte-behandling/src/main/kotlin/vilkaarsvurdering/ektedao/VilkaarsvurderingRepository.kt

```diff
@@ -0,0 +1,447 @@
+package no.nav.etterlatte.vilkaarsvurdering.ektedao
+
+import kotliquery.Row
+import kotliquery.Session
+import kotliquery.queryOf
+import no.nav.etterlatte.common.ConnectionAutoclosing
+import no.nav.etterlatte.libs.common.sak.SakId
+import no.nav.etterlatte.libs.common.tidspunkt.toLocalDatetimeUTC
+import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.libs.common.tidspunkt.toTimestamp
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Delvilkaar
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Vilkaar
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarVurderingData
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Vilkaarsvurdering
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingResultat
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingUtfall
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VurdertVilkaar
+import no.nav.etterlatte.libs.database.tidspunkt
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.YearMonth
+import java.util.UUID
+
+fun <T> Session.hent(
```

Comment:
Trur desse to er definert i database-biblioteket, så du like godt kan bruke dei direkte derifrå?

Reply:
Ja, men for en `TransactionalSession` som autocommiter for deg, her er det kun på `Session` som gjør at det ikke autocommites inne i en `inTransaction`.